Sources/Swift/Integrations/UserFeedback/Configuration/SentryUserFeedbackWidgetConfiguration.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+#if (os(iOS) || os(tvOS)) && !SENTRY_NO_UIKIT
+@_implementationOnly import _SentryPrivate
+import UIKit
+
+/**
+ * Settings for whether to show the widget and how it should appear.
+ */
+@objc public class SentryUserFeedbackWidgetConfiguration: NSObject {
```

Comment:
m: Maybe we should add options to configure the color/format/image of the widget?

Reply:
Maybe! @bruno-garcia wdyt? It's not in any of the other SDKs, we just provide the option to use a fully custom-built button. But theming the stock button makes sense IMO too.

Reply:
Will leave this out for now, can always add it later. Noted for future consideration.